lib/ts/recipe/thirdparty/types.ts

```diff
@@ -109,7 +108,7 @@ export type NormalisedSignInAndUpFeatureConfig = NormalisedBaseConfig & {
     providers: Provider[];
 };
 
-export type GetRedirectionURLContext = AuthRecipeModuleGetRedirectionURLContext;
+export type GetRedirectionURLContext = any;
```

Comment:
why any? shouldn't we type it properly?

Reply:
Add that as a comment.

Reply:
It should technically be `never` but it caused some build issues. However, upon further investigation I could make it work.